### PR TITLE
Replace '$' with '_' in addresses

### PR DESCRIPTION
**Pull request by hanssv**
Solves https://www.pivotaltracker.com/story/show/160254986

### CODE REVIEW

**Comment by hanssv** [comment]
Corresponding code changes [PR1544](https://github.com/aeternity/epoch/pull/1544)

File: epoch/api/oracle_api_usage.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ oracle is created:
 ```
 curl http://localhost:3013/v2/oracles/ok%24EmJyR97vW4jzdcCPCvgjUa8RUmo45E1KnExBum38yz48Frwov
```
Comment (lucafavatella) [inline epoch/api/oracle_api_usage.md]: `%24` is percent-encoding of `$`

File: epoch/api/oracle_api_usage.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Again, we can check that the TTL was extended:
...
```
Comment (lucafavatella) [inline epoch/api/oracle_api_usage.md]: `%24` is percent-encoding of `$`

File: epoch/api/oracle_api_usage.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ We can check the active oracle queries:
 ```
 curl http://localhost:3013/v2/oracles/ok%24EmJyR97vW4jzdcCPCvgjUa8RUmo45E1KnExBum38yz48Frwov/queries
```
Comment (lucafavatella) [inline epoch/api/oracle_api_usage.md]: `%24` is percent-encoding of `$`

File: epoch/api/oracle_api_usage.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ be added to the chain), we see that there is a response for the query:
...
```
Comment (lucafavatella) [inline epoch/api/oracle_api_usage.md]: `%24` is percent-encoding of `$`

**Pull request merged**